.github/workflows/tox.yml

```diff
@@ -0,0 +1,32 @@
+---
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install tox tox-gh-actions
+      - name: Test with tox
+        # Explicitly specifying the testenvs shouldn't really be
+        # necessary here, but for some reason tox invokes no testenvs
+        # at all if invoked as just "tox" from the GitHub Actions
+        # workflow. Until that is fixed, name the testenvs.
+        run: tox -e gitlint,yamllint,build
```

Comment:
Should we gather the artifacts and display them somewhere, to help at reviewing?

Reply:
For now I've only made them downloadable as `build-3.8.zip`, `build-3.9.zip`, and `build-3.10.zip`.